packages/carbon-react/src/components/StructuredList/StructuredList.stories.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { CheckmarkFilled16 } from '@carbon/icons-react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+} from '.';
+import StructuredListSkeleton from './StructuredList.Skeleton';
```

Comment:
```suggestion
import {
  StructuredListWrapper,
  StructuredListHead,
  StructuredListBody,
  StructuredListRow,
  StructuredListInput,
  StructuredListCell,
 StructuredListSkeleton
} from '../StructuredList';
```

Let me know if this works 🤔 Would be great to get them all from the same export